libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectLocation.kt

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+import com.mapbox.geojson.Geometry
+
+/**
+ * RoadObjectLocation
```

Comment:
Could you elaborate here what the purpose of this class?

Reply:
done